td.vue/tests/unit/service/x6/graph/graph.spec.js

```diff
@@ -158,5 +168,14 @@ describe('service/x6/graph/graph.js', () => {
         it('uses the export plugin', () => {
             expect(graphRes.use).toHaveBeenCalledWith(new Export());
         });
+
+        // NEW TEST: wheel event listener added by your feature
```

Comment:
Nit-picky: remove this comment, it doesn't add anything, just an AI artifact. :)